docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -77,6 +77,33 @@ $ sudo usermod -a -G libvirt <username>
 $ newgrp libvirt
 ----
 
+[[kvm-driver-arch-linux]]
+=== On Arch Linux
+
+. Install *libvirt*, *qemu*, *dnsmasq* and *ebtables* on your system:
++
+----
+$ sudo pacman -S libvirt qemu dnsmasq ebtables
+----
+
+. Add yourself to *kvm* and *libvirt* groups:
++
+----
+$ sudo usemod -a -G kvm,libvirt <username>
+----
```

Comment:
Any specific reason user needs to be added to both `kvm` and `libvirt` groups? For other distribution we only add to libvirt group.

Reply:
checked by reinstalling all the packages and doing the configurations, and it is required to be part of both the groups, but still do not know for sure, why the user needs to be added to both the groups. My guess is that, both `kvm` and `libvirt` groups permission is required for running a kvm virtual machine in Arch Linux.